ccViewer/ccViewerApplication.cpp

```diff
@@ -24,13 +24,22 @@
 #include "ccviewer.h"
 #include "ccViewerApplication.h"
 
-
+/*
+ * Initialise the ccViewerApp application
+ * this is version 1.40 alpha, it implements ccApplicationBase
```

Comment:
I don't think we want (or we will remind) to update this line each time we upgrade the version